Website/Website/securedSettings.js

```diff
@@ -51,6 +51,10 @@ module.exports = async function(host) {
                     env.mongoDbUrl = await getSecretOrThrow('mongoDbUrl');
                     env.mongoSharedDbUrl = await getSecretOrThrow('mongoSharedDbUrl').catch(err => env.mongoDbUrl);
                 },
+                ,
+                async function() {
+                    env.kubernetesServices = await getSecret('kubernetesServices');
```

Comment:
by looking at the code here, would be better to have this line here
JSON.parse(env.kubernetesServices);

Reply:
updated.